woocommerce/woocommerce
# Change copy of the payments welcome modal

### All Submissions:






### Changes proposed in this Pull Request:



Closes https://github.com/Automattic/woocommerce-payments/issues/4827.

Updated the copy text in the payments welcome modal according to the P2 comment parYuf-sD-p2#comment-787.

<div>
Before <br/>
<img width="500" alt="image" src="https://user-images.githubusercontent.com/79862886/195063312-d395afcc-93e9-4368-9028-ac6997901070.png">
</div>
<div style="float:left;">
After <br/>
<img width="500" alt="image (1)" src="https://user-images.githubusercontent.com/79862886/195063329-1a6a6fa1-4683-4fe8-89c4-023825e79867.png">
</div>

### How to test the changes in this Pull Request:

1. Checkout the branch `tweak/wcpay-4827-change-copy-of-the-payments-welcome-modal`
2. Up it locally and visit `wp-admin/admin.php?page=wc-admin&path=%2Fwc-pay-welcome-page`
3. Click "No, thanks" and observe the modal is shown.
4. Check that the new copy is correct.

### Other information:




### FOR PR REVIEWER ONLY:
